### PR TITLE
Add extended-const to update script

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by sbc100** [comment]
The result of running the script is in https://github.com/WebAssembly/testsuite/pull/52

File: update-testsuite.sh [hunk 1]
```diff
@@ -103,14 +104,14 @@ failed_repos=
...
 for repo in ${repos}; do
     echo "++ updating ${repo}"
-    if [ "${repo}" = "simd" -o \
-         "${repo}" = "threads" -o \
-         "${repo}" = "exception-handling" -o \
-         "${repo}" = "memory64" -o \
-         "${repo}" = "host-bindings" ]; then
-      branch=main
-    else
+    if [ "${repo}" = "spec" -o \
```
Comment (sunfishcode) [inline update-testsuite.sh:107]: `spec` has now switched to main

**Review by sunfishcode — APPROVED** [review]
Yay for switching the direction of the repos list :-).

File: update-testsuite.sh (hunk 1 above)
Comment (sbc100) [reply update-testsuite.sh]: Done

**Pull request merged**